rules-reviewed/camel3/camel2/java-generic-information.windup.xml

```diff
@@ -110,7 +110,7 @@
                                 href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_generic_information"/>
                         <quickfix type="REPLACE" name="Registry-replace">
```

Comment:
Based on the next comment about having 2 quickfix, the name of this one should change accordingly
```suggestion
                        <quickfix type="REPLACE" name="PropertyPlaceholderDelegateRegistry-replace">
```